src/teststat/BinnedNLLH.cpp

```diff
@@ -116,6 +116,12 @@ BinnedNLLH::AddSystematic(Systematic* sys_){
 }
 
 void
+BinnedNLLH::SetDataSet(DataSet& dataSet_){
```

Comment:
boost::python can only handle pass by reference, it either needs a confusing wrapper function that I still haven't gotten to work within the binding implementation or a simple extra method like this in the class.